### ISSUE TITLE
Cannot Generate Database 

### ISSUE THREAD

**Issue by talk2sohail** [DeepRank/deeprank#209]
**Describe the bug**
While the installation process was perfectly fine, testing the successful installation involved running several files as mentioned in the website or in README.md file. While creating the database ( running the test_generate.py ) It is showing some ```ValueError: Residue numbering not identical in ref and decoy ``` and hence the database cannot be created.

**Environment:**
 Google colab ( hosted runtime ) 
- OS system: Linux ( Ubuntu )
- Version: 20.04
![Screenshot from 2020-10-29 14-23-38](https://user-images.githubusercontent.com/37415268/97546582-89232b80-19c4-11eb-9998-2be547d88619.png)
![Screenshot from 2020-10-29 14-23-58](https://user-images.githubusercontent.com/37415268/97546601-93ddc080-19c4-11eb-9a3f-7965088a5c4d.png)

**Comment by CunliangGeng** [comment]
Thanks for trying our package! 
Could you try to install DeepRank from the ["development" branch](https://github.com/DeepRank/deeprank/tree/development) with `pip install -e .`?  This branch has the latest code that has solved the issues you mentioned. 

We're going to merge the updates to mater branch and release a new version soon.

**Comment by talk2sohail** [comment]
Thank you very much for speedy reply!
I was able to generate the Dataset for testing the installation part. However my need to is to run the model as a whole.
It would be too much to ask from you if you could please draw on your experience and mention the correct  steps to run and train the model as the documentation seems a bit hard to follow.   

Your response is highly solicited.

**Comment by DarioMarzella** [comment]
Hello talk2sohail, thank you for your feedback and sorry for the late answer.

The documentation has been updated in the branch [doc_DM](https://github.com/DeepRank/deeprank/tree/doc_DM). If you clone it, you will be able to find all the documentation HTML files under `docs/_build/html/`.
Could you please have a look at this documentation and see if the procedure seems clearer to you? More specifically, you should use the DeepLearning Tutorial (`docs/_build/html/tutorial_deeplearning.html` ).
Let me know if it is clearer to you, so we can improve it.
